# error_handling-custom_errors Documentation

<!-- BEGIN_PY_DOCS -->
## error_handling-custom_errors.py

### Functions

#### `error_handling_example()`

Raises and handles a custom exception.


<!-- END_PY_DOCS -->